src/components/dependencies/ComplianceCells.js

```diff
@@ -250,7 +250,7 @@ const ComplianceCell = ({
       artefactSummary.artefact.artefact_kind === artefact.kind
       && artefactSummary.artefact.artefact.artefact_name === artefact.name
       && artefactSummary.artefact.artefact.artefact_version == artefact.version
-      && artefactSummary.artefact.artefact.artefact_type == artefact.type
+      && artefactSummary.artefact.artefact.artefact_type === artefact.type
```

Comment:
Do we need to use strict equality operator here, or is it just a measure of precaution?

Reply:
@zkdev : I think it is a good practise to basically _always_ use strict equality-comparison, as the default one is severely broken (see the maaaany memes out there mocking ES for that ;-))

Reply:
![image](https://github.com/user-attachments/assets/40cdfaaf-5e48-4235-af6a-dd66e277e2dd)

Reply:
Actually, this was just some sort of cleanup and not directly related to the extra-id handling (in that: AFAIK, it did not lead to any bugs before).